index.html

```diff
@@ -3,5 +3,7 @@
   <head>
     <title>Welcome to XYZ</title>
   </head>
-  <body></body>
+  <body>
+    <h1>Hey Customer! Welcome to our shop</h1>
```

Comment:
This is good, but could you add a message. "We are ready to serve you" and make the color Red. Thanks.